changelog.d/2573.change.rst

```diff
@@ -0,0 +1,2 @@
+Fixed error in uploading a Sphinx doc with the :code:`upload_docs` command. An html builder will be used. 
+Note: :code:`upload_docs` is deprecated for PyPi, but is supported for other sites -- by :user:`melissa-kun-li`
```

Comment:
FTR PyPI should be spelled with a capital "I"

Reply:
Ah right! Thanks for pointing that out, will note for future.